lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -322,6 +325,16 @@ export type StaticFields = {
     matsHttpEventCount?: number;
     httpVerToken?: string;
     httpVerAuthority?: string;
+
+    /**
+     * WAM bridge flags
+     */
+    wamAllowed?: boolean;
```

Comment:
Let's avoid using "WAM", we chose to use "native" instead to be more agnostic and include future broker support on platforms other than Windows

Reply:
Sounds good. Renamed fields as per your suggestions.